### ISSUE TITLE
Accessibility: V3 Select's options are not being read with Chrome when arrowed through

### ISSUE THREAD

**Issue by danbivins** [department-of-veterans-affairs/vets-design-system-documentation#2610]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened
With the update to the V3 components the Va-Select component's options are not being read by VoiceOver consistently on Chrome.

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/10236001/5f3abf17-51d8-486f-bbd7-f93a879e2364

## What I expected to happen
I expect to hear the options read as I navigate through the component. 

## Reproducing
- Formation version:
- Device:  Mac 14.13.1
- Browser: Chrome Version 122.0.6261.129 (Official Build) (arm64)

Steps to reproduce:

1. Go to [va-select](https://design.va.gov/storybook/?path=/docs/uswds-va-select--default) while using Chrome on a Mac
2. Turn on VoiceOver
3. Navigate to the select component and open so that the options display
4. arrow down and up through the options

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [x] Other - an issue in Production

## Notes
I started to add a defect severity to this ticket but struggled given that it's only occurring in Chrome + Safari. Feel free to add a severity as you see fit.

cc [christinec-fftc](https://github.com/christinec-fftc)

**Comment by caw310** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2610?planning-poker) with Zenhub @rsmithadhoc

**Comment by rsmithadhoc** [comment]
Hi @danbivins, I am not seeing the behavior from your video on my end, the options are reading out. Can you confirm if this is still an issue? Thanks!

**Comment by rsmithadhoc** [comment]
I was going to reach out to Dan on Slack, but it appears they are no longer on the project. I'll close this issue out, as I'm no longer seeing the problem with VoiceOver.
